src/Messages.js

```diff
@@ -700,7 +700,7 @@ export default defineMessages({
     defaultMessage: 'Group description',
   },
   groupDefinition: {
```

Comment:
@aferd sorry, one last request, could you rename also these keys to match the IDs? Thank you :-)

Reply:
oh, forgot to push that up! doing it now.